judgels-backends/jophiel/jophiel-dist/Dockerfile

```diff
@@ -10,6 +10,6 @@ WORKDIR /judgels/jophiel
 EXPOSE 9001 9101
 
 COPY build/distributions .
+COPY init.sh .
 
-ENTRYPOINT ["./service/bin/init.sh"]
-CMD ["console"]
+CMD ["/bin/bash", "-c", "./init.sh ${SERVICE}"]
```

Comment:
Why? I think it will be better for it to be an `ENTRYPOINT`, so that we can just override the `CMD` to be the argument that we want